### DISCUSSION TITLE
Add part manifest

### DISCUSSION THREAD

**Discussion by microbmen** [tjboldt/Apple2-IO-RPi#137] [General]
Is there a parts manifest anywhere?  

I see chip numbers / part numbers printed on the board.  There is the Apple2IORPi.net file, which gives more clarity.  However, I don't see anywhere the value of part C; the capacitors?  
A manifest / BoM would be excellent which lists all the parts, and qty of each needed.  

Thanks,
Mike

**Comment by tjboldt** [comment]
Thanks for the input. I added a bill of materials and fixed the PDF version of the schematic.

**Reply by microbmen** [reply]
Thanks, this looks great!
